compiler/src/dmd/func.d

```diff
@@ -217,6 +217,8 @@ private struct FUNCFLAG
     bool hasAlwaysInlines;   /// Contains references to functions that must be inlined
     bool isCrtCtor;          /// Has attribute pragma(crt_constructor)
     bool isCrtDtor;          /// Has attribute pragma(crt_destructor)
+    bool hasEscapingSiblings;/// Has sibling functions that escape
+    bool computedEscapingSiblings; /// `hasEscapingSiblings` has been computed
```

Comment:
need to update headers, not just `frontend.h`.

Reply:
I don't think LDC or GDC need access to these fields

Reply:
ping @ibuclaw @JohanEngelen

Reply:
it's irrelevant whether LDC or GDC need _access_ to these fields. It's about whether data layout will be different when these are added. This still seems to fit in the 32 bits allocated to the bitfield, which _is_ in frontend.h.

Reply:
so ping @thewilsonator to unblock.